portfolio/src/main/java/PortfolioServer.java

```diff
@@ -23,31 +23,17 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns HTML that contains the page view count. */
-@WebServlet("/page-salutation")
+@WebServlet("/show-series")
 public final class PortfolioServer extends HttpServlet {
 
+    String[] series = new String[]{"Lupin", "Stranger Things", "New Girl", "Grace and Frankie", "Elite", "You", "Control Z", "The Good Place", "The Crown"};
```

Comment:
```suggestion
    String[] series = {"Lupin", "Stranger Things", "New Girl", "Grace and Frankie", "Elite", "You", "Control Z", "The Good Place", "The Crown"};
```
You could also write it like this and not use the `new String[]` constructor, in case you didn't know.

Reply:
Thank you so much! I'll make sure to make these changes on the other files!